spec/features/profiles_spec.rb

```diff
@@ -74,3 +74,36 @@
     end
   end
 end
+
+
+feature 'Websites New' do
+  describe "Form" do
+    before :each do
+      Website.create(
+        name: 'Sample Website',
+        url: 'http://www.sample.com/'
+      )
+      visit '/profiles/new'
+    end
+
+    it "has forms" do
+      expect(page).to have_selector('input#profile_name')
+      expect(page).to have_selector('select#profile_website_id')
+      expect(page).to have_selector('input#profile_url')
+    end
+    it "can not submit form without required fields" do
+      click_on('Add New Profile')
+      expect(page.current_path).to eql new_profile_path
```

Comment:
Based on the language of the spec, this should probably be checking to make sure the submit event never occurred?  or perhaps make sure a POST request didn't happen?  bunch of different ways to verify behavior here... this wouldn't catch a submit and returns the user back to the same url.